integration/setup.go

```diff
@@ -38,9 +39,10 @@ import (
 )
 
 var (
-	portF      = flag.Int("port", 0, "port to use; if 0, in-process FerretDB is used")
-	handlerF   = flag.String("handler", "pg", "handler to use for in-process FerretDB")
-	proxyAddrF = flag.String("proxy-addr", "", "proxy to use for in-process FerretDB")
+	portF       = flag.Int("port", 0, "first system's port for tests; if 0, in-process FerretDB is used")
```

Comment:
"First system" is a little bit unclear. Can we call it something like "target system" instead? I guess it's the main system we want to test, right?

Reply:
Done